package.json

```diff
@@ -72,6 +72,7 @@
 		"redent": "^3.0.0",
 		"shimmer": "^1.2.1",
 		"strip-ansi": "^6.0.0",
+		"tty": "^1.0.1",
```

Comment:
What's this / what's it used for?

Reply:
`tty` is imported at https://github.com/directus/cli/blob/main/src/core/utils.ts#L1 but the library was missing from package.json.

Reply:
That's because it's a node native module like path or fs https://nodejs.org/api/tty.html

Reply:
Did you check what you installed? Even the tty npm package calls it out! https://www.npmjs.com/package/tty

Reply:
Yeah I’m removing it now, it had been included to try to fix things with the other packages but turned out to not be the issue.